src/Specification/AndSpecification.php

```diff
@@ -20,22 +20,22 @@
 final class AndSpecification extends CompositeSpecification implements SpecificationInterface
```

Comment:
Should `implements` be removed here, since it is now inherited from the abstract parent?

Reply:
it would be more eager to do so, will remove those

Reply:
Still going to drop the `implements` from the `final` children?

Reply:
sorry must have missed this one.